spec/features/users/user_deletes_account_spec.rb

```diff
@@ -0,0 +1,36 @@
+# As an authenticated user
+# I want to delete my account
+# So that my information is no longer retained by the app
+
+
+feature 'user deletes account', %Q{
```

Comment:
Use `%` instead of `%Q`.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Use `%Q` only for strings that contain both single quotes and double quotes, or for dynamic strings that contain double quotes.